kitops/modelkit/kitfile.py

```diff
@@ -199,9 +194,7 @@ def load(self, path):
         except yaml.YAMLError as e:
             if mark := getattr(e, "problem_mark", None):
                 raise yaml.YAMLError(
-                    "Error parsing Kitfile at "
-                    + f"line{mark.line + 1}, "
-                    + f"column:{mark.column + 1}."
+                    "Error parsing Kitfile at " + f"line{mark.line + 1}, " + f"column:{mark.column + 1}."
```

Comment:
Does it need a space after the word "line"

Reply:
Eww. I didn't notice how this got reformatted. Will update to make this look better on one line.